module/prover.go

```diff
@@ -16,6 +17,8 @@ import (
 
 var _ core.Prover = (*Prover)(nil)
 
+var IBCCommitmentsSlot = common.HexToHash("1ee222554989dda120e26ecacf756fe1235cd8d726706b57517715dde4f0c900")
```

Comment:
@yoshidan  Can you add a comment `// keccak256(abi.encode(uint256(keccak256("ibc.commitment")) - 1)) & ~bytes32(uint256(0xff))` to the above line?